packages/blade/src/components/Icons/_Svg/ClipPath/ClipPath.d.ts

```diff
@@ -0,0 +1,10 @@
+import type { ReactElement, ComponentType } from 'react';
+
+export type ClipPathProps = {
+  children: ReactElement | ReactElement[];
+  id: string;
+};
+
+const ClipPath: ComponentType<ClipPathProps>;
```

Comment:
why/where do you need this?

Reply:
Multiple icons in `blade-old` were already using this. `EyeOff`, `Success`, `UploadCloud`, etc.